src/boc/Slice.ts

```diff
@@ -55,6 +56,33 @@ export class Slice {
         return this._refsOffset;
     }
 
+    have(bits: number): boolean {
```

Comment:
Grammar issue: should be "has", also method name is ambiguous, I suggest removing it in favor of straightforward comparison with "remainingBits"